core/src/main/java/com/datastax/oss/driver/internal/core/data/DefaultUdtValue.java

```diff
@@ -91,6 +92,39 @@ public ProtocolVersion protocolVersion() {
     return type.getAttachmentPoint().protocolVersion();
   }
 
+  @Override
+  public boolean equals(Object o) {
+    if (!(o instanceof DefaultUdtValue)) return false;
```

Comment:
nit: our style guide requires braces for if statements, even when there is only one statement. I used to omit braces too, but in hindsight I find it slightly less readable.